frontend/src/app/main/ui/workspace/sidebar/options/rows/color_row.scss

```diff
@@ -23,157 +23,161 @@
   gap: $s-2;
   border-radius: $s-8;
   background-color: var(--input-details-color);
-  height: $s-32;
-  width: 100%;
+  block-size: $s-32;
```

Comment:
we are not using block-size

Reply:
I'll change it. 
Anyway, I suggest reconsidering this since it helps the whole team to get used to the terminology inline/block instead of width/height. It has no disadvantages that I can think of, and it can be updated gradually.